contributing/core_team.rst

```diff
@@ -231,7 +231,7 @@ following these rules:
 * **Feature**: For new features and deprecations; Pull requests must be merged
   in the development branch.
 * **Bug**: Only for bug fixes; We are very conservative when it comes to
-  merging older, but still maintained, branches. Read the :doc:`maintenance`
+  merging older, but still maintained, branches. Read the :doc:`/contributing/code/maintenance`
```

Comment:
@OskarStark this might be an interesting one for DOCtor when you have time: we want all doc references to be absolute paths, so the links don't break when moving documents/sections. This has been a rule since forever, but every once in a while one relative path slips in :)